elastic_record.gemspec

```diff
@@ -20,4 +20,5 @@ Gem::Specification.new do |s|
 
   s.add_dependency 'arelastic', '>= 3.4.0'
   s.add_dependency 'activemodel'
+  s.add_dependency 'activesupport'
```

Comment:
With this change, we can now change the line above it to add_development_dependency.